scripts/runningnotes_from_sequencing.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+DESC = """EPP script to summarize sequencing start results to the projects running notes
+"""
+import datetime
+import logging
+import re
+from argparse import ArgumentParser
+
+from genologics.config import BASEURI, PASSWORD, USERNAME
+from genologics.entities import Process  # , Project
+from genologics.lims import Lims
+from write_notes_to_couchdb import write_note_to_couch
+
+from scilifelab_epps.utils.get_epp_user import get_epp_user
+from scilifelab_epps.wrapper import epp_decorator
+
+regex_projectid_line = re.compile(r"^(P\d+)[\s]*:")
+# regex_projectname_line = re.compile(r"^([A-Za-z]+\.[A-Za-z]+_\d{2}_\d{2})[\s]*:")
+TIMESTAMP: str = datetime.datetime.now().strftime("%y%m%d_%H%M%S")
+"""
+Example running note
+
+Comment from Load to Flowcell (NovaSeqXPlus) v1.0 (LIMS) :
+Sequencing started 2025-03-31
+Pool 'POOLNAME' in lane 1, xpM, y% PhiX,
+Pool 'POOLNAME' in lane 2, xpM, y% PhiX,
+1.5B FC=22XDFCRT, on Inst A, 1_3_4_2
+
+/Technician Name
+"""
+
+
+@epp_decorator(script_path=__file__, timestamp=TIMESTAMP)
+def main(args):
+    lims = Lims(BASEURI, USERNAME, PASSWORD)
+    pro = Process(lims, id=args.pid)
+    pro_udfs = {}
+    pools = {}
+    general_comments = []
+    project_specific_comments = {}
+    projects = {}
+
+    for name, value in pro.udf.items():
+        pro_udfs[name] = value
+    # Use get for non mandatory UDFs
+    seq_setup = f"{pro_udfs['Read 1 Cycles']}_{pro_udfs['Index Read 1']}_{pro_udfs.get('Index Read 2', 'x')}_{pro_udfs.get('Read 2 Cycles', 'x')}"
+    inst = f"{pro_udfs['Instrument']} {pro_udfs['Side']}"
+
+    date_started = datetime.datetime.fromisoformat(pro.step.date_started).date()
+
+    pool_artifacts = pro.analytes()[0]
+    for pool_artifact in pool_artifacts:
+        pool_obj = {}
+        for name, value in pool_artifact.udf.items():
+            pool_obj[name] = value
+
+        pool_obj["pool_name"] = pool_artifact.name
+        pools[pool_artifact.id] = pool_obj
+        for sample in pool_artifact.samples:
+            if sample.project.id not in projects:
+                projects[sample.project.id] = [pool_artifact.id]
+            else:
+                projects[sample.project.id].append(pool_artifact.id)
+
+    # The default value for the Comment field in LIMS contains lines that start with '//' which should be filtered out if they are not removed
+    # They could also be used to save info in the step that does not have to be in Genstat
+    for line in pro_udfs.get("Comment").splitlines():
```

Comment:
```suggestion
    for line in pro_udfs.get("Comments").splitlines():
```

The conventional name for the field is "Comments" and I would prefer not to diverge from it ^^

This code change necessitates a LIMS configuration change.

Reply:
Good that we haven't configured LIMS yet 😄